frontend/models/feedback-task-model/global-question/GlobalQuestion.model.js

```diff
@@ -0,0 +1,22 @@
+import { Model } from "@vuex-orm/core";
+
+class GlobalQuestion extends Model {
+  static entity = "globalQuestions";
+  static primaryKey = ["component_type", "question"];
+
+  static fields() {
```

Comment:
Here I'm missing the question name. A value that will be used to link a record response value with the corresponding question. For instance, If we have a record containing the following responses:

```json
{ 
  "question_01": { "value": 3 },
  "question_02": { "value": "NEGATIVE" }
}
```

Then, that means the dataset contains at least 2 questions as follows:
```json
[
  { "id": ..., "name": "question_01", "question": "Rate the output from blablablab...", ... },
  { "id": ..., "name": "question_02", "question": "Classify based on following blablabla...", ... }
]
```

Reply:
To do the connection between question/recordAnswer internally I use the id of the entity GlobalQuestion. This mean I can update the table with the 'name' in the field 'id'.

Reply:
I add the name attribute and remove the primary key.